packages/react-i18n/src/utilities/translate.tsx

```diff
@@ -45,10 +45,12 @@ export interface TranslateOptions<Replacements = {}> {
 }
 
 function numberFormatCacheKey(
-  locale: string,
-  options: Intl.NumberFormatOptions = {},
+  locales?: string | string[],
+  options?: Intl.NumberFormatOptions,
 ) {
-  return `${locale}${JSON.stringify(options)}`;
+  return `${
```

Comment:
I would prefer we move the ternary outside of the interpolation for readability. Any strong opinion?